kivy_ios/recipes/ffpyplayer/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 
 class FFPyplayerRecipe(CythonRecipe):
-    version = "4.2.0"
+    version = "v4.3.5"
     url = "https://github.com/matham/ffpyplayer/archive/{version}.zip"
```

Comment:
I slightly overlooked that change, but don't we usually exclude the `v` from the version so the package pinning won't have to include it?
So I mean `url` could be:
```python
url = "https://github.com/matham/ffpyplayer/archive/v{version}.zip"
```

Reply:
Yes/no ? Otherwise there is no way to go back to a specific archive if not pinned with a v ?

Reply:
🤔 That's a good question.

Pros:
- Users will likely try to pin versions like `ffpyplayer==4.x.x`, as they are used to doing the same when installing things via pip (and etc).

Cons:
- We can't pin a specific hash or tag (if that tag is not prepended by `v`) without changing the URL

Some personal considerations about it + one extra question:
- Recipes are not expected to work fine between multiple versions of the same package (or at least are not tested), so pinning a specific version of a package that needs a recipe to install should be discouraged IMHO.
- Maybe `ffpyplayer` is not the case, but sometimes we need to target a specific tag or hash as a temp workaround (that happened in past with SDL2 and Kivy).
- Maybe we should print a **WARNING** regarding potential unexpected issues while pinning a specific version on a recipe? 

BTW, I'm +1 for keeping the `v` into the version.

Reply:
All good point thanks guys, I leave that to you, I'm fine either ways :+1: